src/str.js

```diff
@@ -1107,7 +1107,7 @@ Sk.builtin.str.prototype.nb$remainder = function (rhs) {
         fieldWidth = Sk.builtin.asnum$(fieldWidth);
         precision = Sk.builtin.asnum$(precision);
 
-        if ((mappingKey === undefined || mappingKey === "") && conversionType != "%") {
+        if ((mappingKey === undefined || mappingKey !== "") && conversionType != "%") {
```

Comment:
what's the reason for this change?

Reply:
oh. my miss. That's work for `dict`, now don't need it.....